pages/docs/running/running-distributed.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Distributed systems
+permalink: running-distributed.html
+keywords: running, distributed, cluster, mainframe, cloud
+summary: "Running preCICE simulations on a distributed system involves additional setup steps which are covered on this page."
+---
+
+## Distributed vs local
+
+A distributed system and a single local system differ in two major ways:
+
+1. The local system can rely on a loopback interface for internal communication.
+   A distributed system requires communication between nodes using a common network.
+2. The local system has unhindered access to the local files.
+   A distributed system requires to use a shared network drive with higher latency.
+
+So, there is additional work involved to set the simulation up.
+
+## Common network interface
+
+preCICE defaults to the loopback interface which will only work on the local system.
+Follow the [instructions on how to configure the communication](configuration-communication)
```

Comment:
Maybe mention here again that this concerns the communication. And specifically only "sockets", right?

Reply:
This is explained on the linked communication page.